<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInput</class>
 <widget class="QWidget" name="DataInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DataInput {
	background-color: rgb(246, 211, 45);
}
#lnl_ip[state = &quot;0&quot;] {
	background-color: red;
}
#lnl_ip[state = &quot;1&quot;] {
	background-color: rgb(38, 162, 105);
}
#lnl_ip[state = &quot;2&quot;] {
	background-color: white;
}
#lnl_port[state1 = &quot;0&quot;] {
	background-color: red;
}
#lnl_port[state1 = &quot;1&quot;] {
	background-color: rgb(38, 162, 105);
}
#lnl_port[state1 = &quot;2&quot;] {
	background-color: white;
}</string>
  </property>
  <widget class="QPushButton" name="bnt_connect">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключиться к серверу</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_nik">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Введите ваш ник</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnl_nik"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_ip">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Введите IP адресс сервера</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnl_ip">
        <property name="inputMask">
         <string>000.000.000.000;_</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="state" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_port">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Введите порт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnl_port">
        <property name="inputMask">
         <string>00000;-</string>
        </property>
        <property name="state1" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
